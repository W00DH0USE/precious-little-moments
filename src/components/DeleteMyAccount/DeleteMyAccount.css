#delete-account-title {
    color: white;
}