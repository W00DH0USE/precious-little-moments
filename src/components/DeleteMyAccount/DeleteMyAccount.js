import React from 'react';
import './DeleteMyAccount.css';

function DeleteMyAccount(props) {
    return (
        <section>
        <div className='modal-wrapper'
                style={{
                    transform: props.show ? 'translateY(0vh)' : 'translateY(-100vh)',
                    opacity: props.show ? '1' : '0'
                }}>
                <div className='modal-header'>
                    <h3 id='delete-account-title'>Delete Account?</h3>
                </div>
                <div className='modal-body'>
                    <p>Account Deletions Cannot be Undone!</p>
                </div>
                <div className="modal-footer">
                    <button className="btn-cancel" onClick={props.close}><span>CLOSE</span></button>
                    <button className="btn-continue" type='button' onClick={props.handleDelete}><span>CONTINUE</span></button>
                </div>
            </div>
        </section>
    )
}

export default DeleteMyAccount;