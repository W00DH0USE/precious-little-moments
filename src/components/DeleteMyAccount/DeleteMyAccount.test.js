import React from 'react';
import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';
import DeleteMyAccount from './DeleteMyAccount';


describe(`DeleteMyAccount component`, () => {
    it('renders without crashing', () => {
        const wrapper = shallow(<DeleteMyAccount />);
        expect(toJson(wrapper)).toMatchSnapshot();
    });
});