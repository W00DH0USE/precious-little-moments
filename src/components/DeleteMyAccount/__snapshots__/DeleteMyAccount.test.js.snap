// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`DeleteMyAccount component renders without crashing 1`] = `
<section>
  <div
    className="modal-wrapper"
    style={
      Object {
        "opacity": "0",
        "transform": "translateY(-100vh)",
      }
    }
  >
    <div
      className="modal-header"
    >
      <h3>
        Delete Account?
      </h3>
    </div>
    <div
      className="modal-body"
    >
      <p>
        Account Deletions Cannot be Undone!
      </p>
    </div>
    <div
      className="modal-footer"
    >
      <button
        className="btn-cancel"
      >
        CLOSE
      </button>
      <button
        className="btn-continue"
        type="button"
      >
        CONTINUE
      </button>
    </div>
  </div>
</section>
`;
